forge-game/src/main/java/forge/game/ability/effects/DigEffect.java

```diff
@@ -421,6 +421,15 @@ else if (!sa.hasParam("NoLooking")) {
                                         sa.getParamOrDefault("WithCounterNum", "1"), sa);
                                 c.addEtbCounter(CounterType.getType(sa.getParam("WithCounter")), numCtr, player);
                             }
+                            if (sa.hasParam("AddTypes")) {
```

Comment:
@Hanmac
Should this be done like `AnimateSubAbility$` instead and prefill LKI?

Reply:
AnimateSubAbility would be more flexible